### ISSUE TITLE
sigsegv when closing "About Emacs" tab

### ISSUE THREAD

**Issue by cswarth** [aquamacs-emacs/aquamacs-emacs#113]
This bug is easy to reproduce. I have tried to be as precise as possible and to also describe circumstances which DO NOT elicit the same crash.

[Attached Crash dump.txt](https://github.com/davidswelt/aquamacs-emacs/files/514728/bug.txt)

```
Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib          0x00007fffeb076dda __pthread_kill + 10
1   libsystem_pthread.dylib         0x00007fffeb161797 pthread_kill + 90
2   libsystem_c.dylib               0x00007fffeaf8d4d7 raise + 26
3   org.gnu.Aquamacs                0x00000001000c2c90 terminate_due_to_signal + 144
4   org.gnu.Aquamacs                0x00000001000dffd3 emacs_abort + 19
5   org.gnu.Aquamacs                0x00000001001bc38c ns_term_shutdown + 124
6   org.gnu.Aquamacs                0x00000001000c2e55 shut_down_emacs + 261
7   org.gnu.Aquamacs                0x00000001000c2c55 terminate_due_to_signal + 85
8   org.gnu.Aquamacs                0x00000001000e1906 deliver_fatal_thread_signal + 134
9   org.gnu.Aquamacs                0x00000001000e28a3 handle_sigsegv + 147
10  libsystem_platform.dylib        0x00007fffeb154bba _sigtramp + 26
11  ???                             000000000000000000 0 + 0
12  org.gnu.Aquamacs                0x00000001001c6b44 -[EmacsView drawRect:] + 84
13  com.apple.AppKit                0x00007fffd3c64f38 -[NSView _drawRect:clip:] + 2276
14  com.apple.AppKit                0x00007fffd3cb51ce -[NSView _recursiveDisplayAllDirtyWithLockFocus:visRect:] + 1753
```
## To reproduce:
1.  Start `Aquamacs` with site customizations, but without personal customizations.
   
   ```
   $ /Applications/Aquamacs.app/Contents/MacOS/Aquamacs  -q
   ```
   
   (You can include most personal customizations, but I'm just trying to narrow down the suspects.)
2. Make the window full screen,
   
   ![screenshot](https://cloud.githubusercontent.com/assets/295428/19169111/cde9012a-8bc7-11e6-8fbe-17f6a592b324.png)
3. Open "About Emacs..."
   
   ![screen shot 2016-10-06 at 1 22 10 pm](https://cloud.githubusercontent.com/assets/295428/19169144/f38406e6-8bc7-11e6-8099-cabdeff6740b.png)
4. Close Emacs - CRASH  (see attached bug traceback)
   You can actually close this any way you like - close just the About Emacs tab, hit `^X^C` to exit emacs, or click on the red window button to close the app.  All will cause the same fault.

You can also do steps 2 and 3 in the opposite order and still see the crash.
## How NOT to reproduce
1. Crash will not occur without site customizations, i.e. this will Not work,
   
   ```
   $ /Applications/Aquamacs.app/Contents/MacOS/Aquamacs  -q -no-site-file
   ```
   
   The site customizations (presumably from aquamacs project) change the way full screen is handled, displaying "about emacs buffer" in a window tab.
2. Crash will not occur unless window in full-screen!

**Comment by HayoBaan** [comment]
Hi Chris, I can't reproduce this on my system (Mac OS X 10.11, Aquamacs 3.3). So something quite specific to your system is causing this. Do other apps work well when going to full screen?

**Comment by treese** [comment]
@cswarth Thanks for the report, especially for the details on how to reproduce it.

With Aquamacs 3.5 on Mac OS 10.14.6 (Mojave), I can't reproduce it. The crash report does not look like others I have received. It's quite possible that this was either fixed between Aquamacs 3.3 and 3.5, or was a library bug in Apple's code that has been fixed.

Since this is old, I'm going to go ahead and close it, but if you still see a problem like this, please reopen the issue.
